source/Calamari.Tests/KubernetesFixtures/KubernetesContextScriptWrapperLiveFixtureEks.cs

```diff
@@ -134,6 +134,7 @@ protected override async Task<Dictionary<string, string>> GetEnvironmentVars(Can
         [Test]
         [TestCase(true)]
         [TestCase(false)]
+        [Ignore("These tests will be transitioned to using a local Kind cluster, rather than a remote cluster")]
```

Comment:
After discussing with @eddymoulton, we realized these tests were running deployments in a single namespace, meaning there was conflicts between test runs.

These actual deployment tests should be run against a local, isolated cluster (such as a Kind cluster). I'll do this work in another PR